faf/api/clan_schema.py

```diff
@@ -14,6 +14,7 @@ class ClanSchema(Schema):
     clan_tag = fields.String()
     clan_leader_id = fields.Integer()
     clan_founder_id = fields.Integer()
+    clan_tag_color = fields.String()
```

Comment:
Have we ever decided whether FAF is en_US or en_GB? Your branch is named `clan_tag_colours` but this field is called `clan_tag_color` (mind the missing `u` ;-))